client/app/bundles/course/level/pages/Level/LevelView.scss

```diff
@@ -0,0 +1,13 @@
+// scss-lint:disable ImportantRule
+.deleteButtonCell {
+  padding-left: 0 !important;
```

Comment:
I'm not too keen to declare `!important` so freely as it overwrites global styles.

Reply:
We use `css-modules` so the CSS is scoped by default. The generated class name for the table header this ends up going to is something like `app-bundles-course-level-pages-Level-___LevelView__deleteButtonCell___2mcr4`.

The `!important` is necessary to override the styles assigned by the material-ui library on the `TableHeaderColumn` element.

Reply:
Ok I had to take a look at the rendered HTML to figure out what is going on. @jeremyyap correct me if I am wrong as well, but what happens is: 
 - We have a Material UI theme which [applies inline styles to all child components rendered within](http://www.material-ui.com/#/customization/styles).
 - To overwrite the theme styles, we can either: (i) overwrite with our own inline styles, or (ii) if we use css-modules, declare `!important` in our css. 

Personally, I am not too keen on having `!important` declared so many times unless absolutely necessary, but it [makes maintenance of CSS really hard](https://stackoverflow.com/a/3706876), particularly cause it is very hard to debug.

Would suggest placing it as an styles object within the file since it is not shared yet.

Reply:
I'm ok with that too. It was done like that for code style consistency with programming test case creation, where `deleteButtonCell` is also used.